src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -32,28 +32,35 @@
 #define ENABLE_OTA_TESTS 1
 #endif
 
-// TODO: Disable test005_DoBDXTransferAllowUpdateRequest until PR #26040 is merged.
+// TODO: Disable test006_DoBDXTransferAllowUpdateRequest,
+// test007_DoBDXTransferWithTwoOTARequesters and
+// test008_DoBDXTransferIncrementalOtaUpdate until PR #26040 is merged.
 // Currently the poll interval causes delays in the BDX transfer and
 // results in the test taking a long time.
-#ifdef ENABLE_TEST_005
-#undef ENABLE_TEST_005
+#ifdef ENABLE_TESTS
+#undef ENABLE_TESTS
 #endif
 
 static const uint16_t kPairingTimeoutInSeconds = 10;
 static const uint16_t kTimeoutInSeconds = 3;
 static const uint16_t kTimeoutWithUpdateInSeconds = 60;
 static const uint64_t kDeviceId1 = 0x12341234;
 static const uint64_t kDeviceId2 = 0x12341235;
+static const uint64_t kDeviceId3 = 0x12341236;
 // NOTE: These onboarding payloads are for the chip-ota-requestor-app, not chip-all-clusters-app
 static NSString * kOnboardingPayload1 = @"MT:-24J0SO527K10648G00"; // Discriminator: 1111
 static NSString * kOnboardingPayload2 = @"MT:-24J0AFN00L10648G00"; // Discriminator: 1112
+static NSString * kOnboardingPayload3 = @"MT:-24J0CEK01L10648G00"; // Discriminator: 1113
 
 static const uint16_t kLocalPort = 5541;
 static const uint16_t kTestVendorId = 0xFFF1u;
 static const uint16_t kOTAProviderEndpointId = 0;
+static const uint64_t kOTAProviderNodeId = 0x12341236;
```

Comment:
This is fairly confusing.  This is not in fact the node id of any OTA provider, right?  In fact, it's the node id of one of the OTA requestors.

I think we should do a few things here, if we want this node id to point off to something that is not a provider:

1.  This should not be the id of any actual device involved.
2. We should name it to make it clear what it is: the id of a nonexistent device.

Reply:
It should be a different number. i didn't update this when i added the device 3. i will just use a 12341300 number for this. This is for testing the default provider id. when we write to the default ota provider we need to check if the same is returned. its some dummy data to set. i will add comments to indicate that

Reply:
i just removed the test to check if we can update the default OTA provider. if it makes sense to add to MTR Device will do so in another PR. Let me know if you think it adds value to MTRDevice tests and i will file an issue to track.